data/categories.txt

```diff
@@ -5345,6 +5353,9 @@ sk:Nevestinec|verejný dom
 sw:Danguro
 fa:فاحشه خانه|جنده خانه
 
+amenity-love_hotel|@category_hotel
```

Comment:
LGTM if @vng is OK with categorizing it.

Reply:
Love hotel is still a hotel :)